addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -601,7 +602,10 @@ var KanbanRenderer = BasicRenderer.extend({
                 e.preventDefault();
                 break;
             case $.ui.keyCode.UP:
-                this._focusOnPreviousCard(e.currentTarget);
+                const previousFocused = this._focusOnPreviousCard(e.currentTarget);
+                if (!previousFocused) {
+                    this.trigger_up('focus_searchview');
```

Comment:
The `navigation_move` event only works between the records. It can be used to get inside the view as it will select the first record, but it won't go any higher than it. The choices were to either change the original function or to add one. I thought the latter was the cleanest.

Reply:
Updated